promitor-agent-scraper/templates/prometheusrule.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.prometheusRule.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "promitor-agent-scraper" . }}
+{{- with .Values.prometheusRule.namespace }}
+  namespace: {{ . }}
+{{- end }}
+  labels:
```

Comment:
Can you align with the other resources here please?

https://github.com/promitor/charts/blob/main/promitor-agent-scraper/templates/deployment.yaml#L7

Reply:
Sure. That makes sense.